components/chainhook-cli/src/config/generator.rs

```diff
@@ -36,6 +39,9 @@ max_number_of_processing_threads = 16
 max_number_of_networking_threads = 16
 max_caching_memory_size_mb = 32000
 
+# If you don't require historical data for your predicates or wish to avoid downloading 
```

Comment:
The TSV file is not downloaded every time. It is downloaded once, and future runs use that cached version until a new one is available.